tools/cold-backup/run.sh

```diff
@@ -30,109 +30,94 @@ FILE_NAME=""
 CUSTOM_CM=""
 CUSTOM_SE=""
 
-start_cluster() {
-  if [[ -n $NAMESPACE && -n $CLUSTER_NAME ]]; then
-    echo "Starting cluster $CLUSTER_NAME"
-    CLEANUP=false 
-    local zoo_ss && zoo_ss="$(kubectl -n "$NAMESPACE" get sts "$CLUSTER_NAME"-zookeeper -o name --ignore-not-found)"
-    if [[ -n $zoo_ss ]]; then
-      kubectl -n "$NAMESPACE" scale "$zoo_ss" --replicas "$ZK_REPLICAS"
-      wait_for "condition=Ready" "pod -l strimzi.io/name=$CLUSTER_NAME-zookeeper"
-    fi
-    local kafka_ss && kafka_ss="$(kubectl -n "$NAMESPACE" get sts "$CLUSTER_NAME"-kafka -o name --ignore-not-found)"
-    if [[ -n $kafka_ss ]]; then
-      kubectl -n "$NAMESPACE" scale "$kafka_ss" --replicas "$KAFKA_REPLICAS"
-      wait_for "condition=Ready" "pod -l strimzi.io/name=$CLUSTER_NAME-kafka"
-    fi
-    local co_deploy && co_deploy="$(kubectl -n "$NAMESPACE" get deploy strimzi-cluster-operator -o name --ignore-not-found)"
-    if [[ -n $co_deploy ]]; then
-      kubectl -n "$NAMESPACE" scale "$co_deploy" --replicas 1
-      wait_for "condition=Ready" "pod -l strimzi.io/kind=cluster-operator"
-    fi
-    wait_for "condition=Ready" "pod -l strimzi.io/name=$CLUSTER_NAME-kafka"
+error() {
+  echo "$@" 1>&2
+  exit 1
+}
+
+check_kube_conn() {
+  kubectl version -o yaml --request-timeout=5s 1>/dev/null
+}
+
+# wait_for "condition=Ready" "kafkas.kafka.strimzi.io/my-cluster" "my-namespace"
+# wait_for "condition=Ready" "pod -l strimzi.io/kind=cluster-operator" "my-namespace"
+# wait_for "delete" "kafkas.kafka.strimzi.io/my-cluster" "my-namespace"
```

Comment:
Is this an example how to use it? Or some left over code? If it is left here intentionally, some explanation might be good.

Reply:
Yes, these examples are intentionally left. Let me improve the comment.